add precision about shared plans and accessLogs (#452)
CleverCloud/doc.clever-cloud.com: commons/tools/metrics.md

<div class=“panel panel-warning”>
  <div class=“panel-heading”>
    <h4>WARNINGS</h4>
  </div>
  <div class=“panel-body”>
  <ul>

>     <li>Shared plan addons do not provide access logs</li>

Reviewer: ```suggestion
    <li>Add-ons on shared plans (usually DEV plans) do not provide access logs</li>
```

>     <li>In case of a direct access to addons. They is no access logs recorded</li>

Reviewer: ```suggestion
    <li>There are no recorded access logs in case of a direct access to an add-on</li>
```
